depmap-compute/depmap_compute/slice.py

```diff
@@ -38,3 +38,26 @@ def decode_slice_id(slice_id) -> tuple[str, str, str]:
     slice_type = unquote(parts[3])  # "label", "entity_id", or "transpose_label"
 
     return dataset_id, dimension_identifier, slice_type
+
+
+def slice_id_to_slice_query(slice_id: str) -> SliceQuery:
+    """Take a legacy slice ID string and convert it to the newer slice query format."""
+    dataset_id, dimension_identifier, slice_type = decode_slice_id(slice_id)
+
+    # Slice query identifier types use different (more descriptive) terminology
+    if slice_type == "label":
+        slice_query_identifier_type = "feature_label"
+    elif slice_type == "transpose_label":
+        # "transpose_label" identifiers are always depmap IDs
```

Comment:
It could also reference a screen ID like this:
https://cds.team/depmap/data_explorer_2/unique_values_or_range?slice_id=slice/breadbox%2F30e1e790-44a3-4af4-9b3b-0efee94c053e/SC-000001.KY01/transpose_label

Reply:
That's not exactly relevant to this PR (I don't think you can access screen-level data through the Genomic Associations tool) but this function looks pretty generic :)